parcel-react-demo/src/Components/CallbackFn/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Button from "../../Button/Button";
+import styles from "./style";
+
+import { useState } from "react";
+
+const TodoForm = ({ addTodo }) => {
+    console.log(TodoForm);
+
+    const [input, setInput] = useState("");
+    // console.log(todos);
+    // console.log(deleteTodo);
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        addTodo(input);
+        setInput("");
+    };
+
+    console.log(input);
+    return (
+        <div style={styles.MainContainer}>
```

Comment:
prefer to write css selector in lowercase letter

Reply:
okay